src/dmd/dclass.d

```diff
@@ -194,7 +194,7 @@ enum ClassKind : int
  */
 extern (C++) class ClassDeclaration : AggregateDeclaration
 {
-    extern (C++) __gshared
+    extern (C++) static __gshared
```

Comment:
It's already `__gshared`. DMD should emit a warning when both are used as this doesn't make any sense.

Reply:
I was following the example here: https://github.com/dlang/dmd/blob/e17992f4b7707b6ce6ba26921cb7420732fe5afd/src/dmd/dmodule.d#L312

But, I'll change it.